modules/perception/obstacle/camera/visualizer/common/gl_raster_text.cc

```diff
@@ -339,6 +339,23 @@ void GLRasterText::print_string(const char *s) {
   glPopAttrib();
 }
 
+void GLRasterText::print_multiline(
+    const char *s,
```

Comment:
why is this file written in c style? It is better to use c++ to keep consistent with other parts in this repo.

Reply:
"c style" means the function name ->PrintMultiLines? if true, I will fix print_multiline() and print_string() together.

Reply:
I mean thing like using char* s to represent string and etc.

Reply:
It would be good if you can also fix the functions names as google coding style generally requuires starting with a capital letter and have a capital letter for each new word